doc/enterprise-shield/o_enterprise_shield_checklist.md

```diff
@@ -0,0 +1,55 @@
+
+![Gateway Topology with Ports Open to the Trusted Network](../images/f-dmz-trusted-top.png)
+
+**Figure 1: A KAAZING Gateway Topology with Ports Open to the Trusted Network**
+
+Using Enterprise Shield™, you can close all inbound ports in your firewall, thus closing the entry points available to untrusted users and eliminating your attack surface vulnerability.
+
+To implement Enterprise Shield™, you configure a Gateway in the DMZ, which receives a reverse connection from within the trusted network. With this architecture, a client can talk to the back-end service or message broker through a firewall. Another benefit of Enterprise Shield™ is that your architecture remains valid, without requiring changes. For example, neither the client nor the back-end service or message broker are aware of the reverse connection because it is completely transparent to the rest of the architecture. Clients that are outside the firewall connect as usual to the DMZ Gateway.
+
+![Simple Topology Showing a Reverse Connection](../images/f-dmz-trustednetwork-860-02.png)
+
+**Figure 2: Closed Inbound Ports in an Enterprise Shield™ Topology**
+
+Adding Enterprise Shield™ to your architecture requires only a few simple modifications. Instead of a single Gateway in the DMZ, you add another Gateway to your trusted network. Then, you only need to make a few changes to the two Gateway configurations to reverse the connection. Other parts of the architecture, such as the client and the back-end service or message broker, observe no apparent differences between a configuration with Enterprise Shield™ or one without, making the reverse connection completely transparent to the endpoints of the configuration.
+
+With this architecture in place, you can close all inbound ports of your firewall, thus providing maximum security and zero attack vectors for malicious users seeking to exploit ports in your firewall.
+
+See [Common Enterprise Shield™ Use Cases](c_enterprise_shield_use_cases.md) to gain more insight into how you might use Enterprise Shield™. This topic describes topologies that vary depending on the level of security, such as configuring Enterprise Shield™ on a single pair of Gateways, on Gateways that are clustered for high availability, whether network transports are used, and so on.
+
+[Walkthrough: Configure Enterprise Shield™](p_enterprise_shield_config.md) provides a detailed walkthrough describing how to configure KAAZING Gateway to protect your enterprise by using a reverse connection to initiate the connection from the Gateway in the internal trusted network to a DMZ Gateway.
+
+**Note:** Before you configure Enterprise Shield™ you should have a basic understanding of the following topologies:
+
+-   Standard Gateway topology:
```

Comment:
This is concepts. In the "About" section we want to make sure the reader understands a standard GW topology and what it means to change the configuration to have forward and reverse connections. We want to give them some definitions before they jump over to start configuring ES.